src/core/modules/LandscapeMgr.cpp

```diff
@@ -1096,6 +1174,7 @@ void LandscapeMgr::onTargetLocationChanged(const StelLocation &loc)
 				setFlagCardinalPoints(conf->value("viewing/flag_cardinal_points", true).toBool());
 				setFlagOrdinalPoints(conf->value("viewing/flag_ordinal_points", true).toBool());
 				setFlagOrdinal16WRPoints(conf->value("viewing/flag_16wcr_points", false).toBool());
+				setFlagOrdinal32WRPoints(conf->value("viewing/flag_32wcr_points", false).toBool());
```

Comment:
what does the wcr actually mean?

Reply:
Also, it may make sense to change the whole interface here and use just one value "cardinal_points" with values

0: none
1: cardinal (NESW)
2: intercardinal
3: secondary
4: tertiary
5: Rhumb number (I am not sure about the English terminology. "Strichzahl" in German.)

Likewise, the GUI may then have a Combobox.

With setting "rhumb" the display should also show different partitions. Either just the rhumb ticks, or both, degrees and rhumb tick?

This may happen in this PR or in another one.

Reply:
This also needs (in any case) an update in the SUG.

Reply:
> Also, it may make sense to change the whole interface here and use just one value "cardinal_points" with values
> 
> 0: none 1: cardinal (NESW) 2: intercardinal 3: secondary 4: tertiary 5: Rhumb number (I am not sure about the English terminology. "Strichzahl" in German.)
> 
> Likewise, the GUI may then have a Combobox.
> 
> With setting "rhumb" the display should also show different partitions. Either just the rhumb ticks, or both, degrees and rhumb tick?
> 
> This may happen in this PR or in another one.

Compass marks already exists and combobox is bad idea here IMHO (just a lecturer point of view)

Reply:
Why is a single ComboBox with 6 possible values worse than 4 checkboxes? 

Combobox labeling: Compass Marks

Values:

none          
4 (Tooltip: cardinal) 
8 (Tooltip: cardinal + intercardinal)
16 (Tooltip: up to secondary intercardinal) 
32 (Tooltip: up to tertiary intercardinal partitions found in navigation)
Rhumb (Tooltip: Rhumb number used in navigation) 

Not strictly required now, but please think of it.

Reply:
Currently any lecturer can change the number of compass labels via shortcuts in any time of lecture. To change the values via combobox user/lecturer should use GUI (RCP as an optional UI) - this is not very good behaviour for lecturers.

Reply:
Hm, OK. But as we always say, these could be done with 1-line scripts... I think of typical users who may want to see less GUI elements. 
BTW, the 32 option should also go into the RCP (both files!) before merge of this PR.

Reply:
Good point for RCP!

Reply:
OK. Now I think all that required to be done is a new Figure 4.13 and a line in D.1.25.